src/components/specimen/components/contentBlocks/Assertions.tsx

```diff
@@ -0,0 +1,70 @@
+/* Import Dependencies */
+import { Card, Row, Col } from 'react-bootstrap';
+import classNames from "classnames";
+
+/* Import Store */
+import { useAppSelector } from "app/hooks";
+import { getSpecimen } from "redux/specimen/SpecimenSlice";
+
+/* Import Components */
+import PropertiesTable from './PropertiesTable';
+
+
+/* Props Typing */
+interface Props {
+    ShowWithAnnotations: Function
+};
+
+
+const Assertions = (props: Props) => {
```

Comment:
Ok so this are only specimen level assertions correct?

Reply:
Yes, that is true, should other assertions be shown here? I think I currently show the other assertions, of occurrences for example, under the occurrence tab.

Reply:
No I think, that is the best approach :+1: